src/components/Search/OmniBox.js

```diff
@@ -80,6 +80,10 @@ export class OmniBox extends Component {
     return isEmpty(query) && isEmpty(params);
   };
 
+  doSearchFromClickEvent = (e, { content }) => {
```

Comment:
Why is the { content } for?

Cannot you just leave this empty?

doSearch knows how to get the query...